.github/workflows/belt-hash.yml

```diff
@@ -59,3 +59,17 @@ jobs:
       - run: cargo test --no-default-features
       - run: cargo test
       - run: cargo test --all-features
+
+  # Build iterate all no_std feature combos
+  build-nostd-features:
```

Comment:
I think you could probably put this under the build job (which is probably more accurately named `no_std` or `build-no_std`), which would reuse intermediate artifacts